### PR TITLE
[FIX] Release - Potential error message from missing transactions details

### PR DESCRIPTION
**Pull request by andreahaku**
**Description**
Fixes potential error from missing transaction details:

image

**Checklist**

* [ ] There is a related GitHub issue
* [ ] Tests are included if applicable
* [ ] Any added code is fully documented

**Issue**

No related issues.

### CODE REVIEW

File: app/components/UI/TransactionElement/TransactionDetails/index.js [hunk 1]
```diff
@@ -179,12 +179,12 @@ class TransactionDetails extends PureComponent {
...
 			chainId,
 			transactionDetails,
-			transactionObject,
 			transactionObject: { status, time, transaction },
 		} = this.props;
 		const renderTxActions = status === 'submitted' || status === 'approved';
 		const { rpcBlockExplorer } = this.state;
-		return (
+
+		return transactionDetails ? (
```
[the remark this answers is not in the document]
Comment (Fatxx) [reply app/components/UI/TransactionElement/TransactionDetails/index.js:187]: What if we remove defaultProps and do a check `if (!transactionDetails) return <>Unavailable transaction data.</>` and we use optional chaining `.?` for object properties.

I'm just thinking this out of my head but tell me your thoughts.

[the remark this answers is not in the document]
Comment (andreahaku) [reply app/components/UI/TransactionElement/TransactionDetails/index.js:187]: > What if we remove defaultProps and do a check `if (!transactionDetails) return <>Unavailable transaction data.</>` and we use optional chaining `.?` for object properties.
> 
> I'm just thinking this out of my head but tell me your thoughts.

Yes this is also another way but then we need some design for it. I started using optional chaining but then there were too many things depending on `transactionDetails` (pretty much everything) and decided to just wrap everything into an if statement

[the remark this answers is not in the document]
Comment (andreahaku) [reply app/components/UI/TransactionElement/TransactionDetails/index.js:187]: > 

You're right. Not sure which way to go but probably I'll take out 'defaultProps' and keep the if statement wrapping. Makes more sense. Thanks

**Review by gantunesr — APPROVED** [review]
lgtm

**Review by Fatxx — APPROVED** [review]

**Pull request merged**